.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,122 @@
+on:
+  workflow_call:
+    inputs:
+      instance:
+        required: true
+        type: string
+
+name: Govulncheck check
+jobs:
+  govulncheck:
+    runs-on: ${{ inputs.instance }}
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install golang
+        run: |
+          ./tests/install_go.sh -f -p
+          echo "/usr/local/go/bin" >> "$GITHUB_PATH"
```

Comment:
Our convention is to prefer putting braces around variable references even when not strictly required, so if you could update these in your workflow that would be appreciated